install_config/install/prerequisites.adoc

```diff
@@ -623,8 +623,16 @@ xref:../../install_config/persistent_storage/persistent_storage_iscsi.adoc#insta
 === SELinux
 
 Security-Enhanced Linux (SELinux) must be enabled on all of the servers before
-installing {product-title} or the installer will fail. Also, configure
-`*SELINUXTYPE=targeted*` in the *_/etc/selinux/config_* file:
+installing {product-title}. 
+
+[IMPORTANT]
+====
+Without SELinux enabled, the installer will fail and there is an increased risk
+of a container accessing the contents of another. Also, you will not have a true
+multi-tenant system.
+====
```

Comment:
@sferich888 done. I'll wait for Eduardo's final ack in the BZ then move forward with this. Thanks!